### PR TITLE
Add deZem documentation to cloud-integrations

### PR DESCRIPTION
**Pull request by deZemGmbH**
#### Summary
Add deZem documentation to cloud-integrations.
...

#### Changes

- Add directory `dezem` to `integrations/cloud-integrations` containing `_index.md` and image files

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by deZemGmbH** [comment]
Unfortunately I wasn't able to test the search bar. I could only test that the doc page looks like it should.

screenshot

**Comment by philippdzm** [comment]
Unfortunately I wasn't able to test the search bar. I could only test that the doc page looks like it should.

screenshot

**Review by johanstokking — COMMENTED** [review]
Thanks, one suggestion to integrate the copy a bit better in these docs.

File: doc/content/integrations/cloud-integrations/dezem/_index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "deZem"
+description: "deZem IoT data management system"
+---
+
+The [deZem IoT data management system](https://www.dezem.de/en/data-acquisition/lorawan/) facilitates an increased efficiency and productivity of plants, processes and infrastructure. To this end we provide a modular solution to collect sensor data from very heterogeneous sources on a single platform, and a large variety of online analysis tools. As LoRaWAN devices are great for effortless data acquisition in the field, the deZem system now also integrates with TheThingsStack V3.<br/>
```
[the remark this answers is not in the document]
Comment (philippdzm) [reply doc/content/integrations/cloud-integrations/dezem/_index.md]: Thanks, @johanstokking, good proposition. I added it to this pull request.

File: doc/content/integrations/cloud-integrations/dezem/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+![deZemAd TheThingsStack import interface](lorawan-and-dezem.svg)
+
+## Prerequisites
+
+1. deZem account
+2. User rights for deZemAd's technical tree
+
+## Setup
+
+In the technical tree of deZemAd, create a new device of type 'LoRaWAN TTS interface'.
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/dezem/_index.md:17]:
```suggestion
In the technical tree of deZemAd, create a new device of type **LoRaWAN TTS interface**.
```

File: doc/content/integrations/cloud-integrations/dezem/_index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+![deZemAd TTS import interface](dezem-ad-lorawan-import-interface.png)
+
+Configure the same API key in deZemAd and in deZem's Webhook template.
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/dezem/_index.md:21]:
```suggestion
Configure the same **API key** in deZemAd and in deZem's Webhook template.
```

File: doc/content/integrations/cloud-integrations/dezem/_index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+![deZem Webhook Template](dezem-webhook-template.png)
+
+## Analyze and Visualize
+
+Create analyses and visalisations for use in production.
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/dezem/_index.md:27]:
```suggestion
Create analyses and visualisations for use in production.
```

**Review by nejraselimovic — APPROVED** [review]

**Comment by nejraselimovic** [comment]
@philippdzm this is great! Please take a look at my suggestions above, I'll merge this as soon as it is approved.

**Review by johanstokking — APPROVED** [review]

**Pull request merged**